src/SearchWrapper.cpp

```diff
@@ -190,21 +180,21 @@ searchWrapper(Rcpp::CharacterVector & dbpath, Rcpp::List & enquireList, Rcpp::Li
 	    output[str] = rowVec;
 	}
 
-	bool displaySpies = true;
-	if (enquireList.containsElementNamed("viewSpies")) displaySpies = Rcpp::as<bool>(enquireList["viewSpies"]);
-
-	if (displaySpies) {
-	    // Fetch and display the spy values
-	    for (std::vector<Xapian::ValueCountMatchSpy *>::iterator itr = spy.begin(); itr != spy.end(); ++itr) {
-		Rcpp::Rcout << "new spy" << std::endl;
-		Xapian::ValueCountMatchSpy * aSpy = *itr;
-		for (Xapian::TermIterator facet = aSpy->values_begin(); facet != aSpy->values_end(); ++facet) {
-		    Rcpp::Rcout << "Facet: " << *facet << "; count: " << facet.get_termfreq() << std::endl;
-		}
+	if (returnSpy) {
+	    Rcpp::List outputSpy;
+	    std::string aStr;
+	    for (Xapian::TermIterator facet = spy->values_begin(); facet != spy->values_end(); ++facet) {
+		aStr = *facet + "; count: ";
+		int termFreq = facet.get_termfreq();
+		stringstream convert;
+		convert << termFreq;
+		aStr += convert.str();
+		outputSpy.push_back(aStr);
 	    }
-	}
+	    return outputSpy;
+	} else
+	    return convertToDataFrame(output);
 
-	return convertToDataFrame(output);
```

Comment:
Is it possible to get access to both the spies and the output? That would be fairly common usage, I think.